cmd/bigsky/main.go

```diff
@@ -292,6 +294,8 @@ func runBigsky(cctx *cli.Context) error {
 	signals := make(chan os.Signal, 1)
 	signal.Notify(signals, syscall.SIGINT, syscall.SIGTERM)
 
+	// TODO: set slog default from param/env
```

Comment:
I think this will be an unexpected degradation from current config, so I would recommend setting this up for relay (bigsky) specifically. There is some example code in `palomar`'s `main.go`.

Reply:
I have a second PR coming with this